examples/docker/stable_diffusion/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Note: this file is from https://github.com/AbdBarho/stable-diffusion-webui-docker/blob/master/build/Dockerfile 
+
+# syntax=docker/dockerfile:1
+
+FROM continuumio/miniconda3:4.12.0
+
+
+RUN conda install python=3.8.5 && conda clean -a -y
+RUN conda install pytorch==1.11.0 torchvision==0.12.0 cudatoolkit=11.3 -c pytorch && conda clean -a -y
+RUN git clone https://github.com/hlky/stable-diffusion.git && cd stable-diffusion && git reset --hard ff8c2d0b709f1e4180fb19fa5c27ec28c414cedd
+RUN conda env update --file stable-diffusion/environment.yaml --name base && conda clean -a -y
+
+# fonts for generating the grid
+RUN apt-get update && apt install fonts-dejavu-core rsync -y && apt-get clean
+
+# Note: don't update the sha of previous versions because the install will take forever
+# instead, update the repo state in a later step
+RUN cd stable-diffusion && git pull && git reset --hard c5b2c86f1479dec75b0e92dd37f9357a68594bda && \
+  conda env update --file environment.yaml --name base && conda clean -a -y
+
+# download dev UI version, update the sha below in case you want some other version
+# RUN <<EOF
+# git clone https://github.com/hlky/stable-diffusion-webui.git
+# cd stable-diffusion-webui
+# # map to this file: https://github.com/hlky/stable-diffusion-webui/blob/master/.github/sync.yml
+# git reset --hard 49e6178fd82ca736f9bbc621c6b12487c300e493
+# cp -t /stable-diffusion/scripts/ webui.py relauncher.py txt2img.yaml
+# cp -t /stable-diffusion/configs/webui webui.yaml
+# cp -t /stable-diffusion/frontend/ frontend/*
+# cd / && rm -rf stable-diffusion-webui
+# EOF
```

Comment:
If not used, should we just remove this? Or maybe make the comments more descriptive - right now its not clear what this code block is for.